src/components/EmployeesList.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+import { fetchEmployees } from '../redux/employees/actions';
+import { employeesSelector } from '../redux/employees/selectors';
+import Employee from './Employee';
+import Error from './Error';
+import Loader from './Loader';
+
+const EmployeesList = () => {
+  const { employees, error, loading } = useSelector(employeesSelector);
+  const { id } = useParams() as any;
+  //const { id } = useParams<{id?: string}>();
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchEmployees(id));
```

Comment:
seems like id should be in dependencies. Because if id of employee changes, useEffect won't be triggerred